### DISCUSSION TITLE
Recipients column format - please bring back the previous format of the Recipient column

### DISCUSSION THREAD

**Discussion by Arpi** [anonaddy/anonaddy#430] [Feature Requests]
Dear Will, 

It was better previously, when it was possible to see even without clicking on a button to view the recipients. Now you have to open them one by one to see the recipient(s) of aliases. 
I have quite a few aliases, and sometimes I just need to quickly find those that do not have the default recipient. 
Previously it was better because it was possible to see (or even find it on the page with Ctrl+F) all the aliases that do not have the default recipient set. 
Please consider changing it back, or if that is not desirable, then add another column with all the recipients displayed. 

Maybe adding new column(s) could be accompanied with the possibility to selectively turn on/off any (or just the extra) columns in the Settings. 

Thanks for this brilliant service, 
All the best, 
Arpad

**Comment by willbrowningme** [comment]
Thanks, the change is a temporary one while I figure out how to better optimise the database queries and relationships. It was simply taking too long to load the page for many users. 

I will bring it back in the new user interface.

**Comment by knstntnb** [comment]
I was looking for a comment or ticket for that one, because I find that new (temporary) design very "not-handy", just spend some time searching for aliases connected with one particular mail-address. :)

Just a suggestion: you could provide an easy overview on the alias-page, where I can see custom-recipient/domain/account (as previously), which was handy enough as well as an overview in the recipient-page to see all aliases, which are attached to that particular "regular mail", where I can switch to the alias for editing.
I think what would be helpful for some people with multiple mail-address.

**Comment by willbrowningme** [comment]
I brought this back a while ago, hopefully you'll be able to notice some loading time improvements too.
